rest-api-spec/src/main/resources/rest-api-spec/api/_global.json

```diff
@@ -0,0 +1,29 @@
+{
+  "description": "Parameters that are accepted by all API endpoints.",
+  "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html",
+  "params": {
+    "pretty": {
+      "type": "boolean",
+      "description": "Pretty format the returned JSON response.",
+      "default": false
+    },
+    "human": {
+      "type": "boolean",
+      "description": "Return human readable values for statistics.",
+      "default": true
+    },
+    "error_trace": {
+      "type": "boolean",
+      "description": "Include the stack trace of returned errors.",
+      "default": false
+    },
+    "source": {
+      "type": "string",
+      "description": "The URL-encoded request definition. Useful for libraries that do not accept a request body for non-POST requests."
+    },
+    "filter_path": {
```

Comment:
I sense that it's unsupported on your because you don't run REST tests? I guess we have to decide whether the spec are the clients spec or the es spec. I think it's the client spec. Why don't you support this parameter like all the other clients do? I am doing the same with the low level java REST client, support for `ignore` is now in the test code but will be moved to the prod code asap.